back/engines/commercial/machine_translations/spec/services/machine_translations/machine_translation_service_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe MachineTranslations::MachineTranslationService do
+  let(:service) { described_class.new }
+
+  describe 'translate' do
+    before do
+      stub_easy_translate!
+    end
+
+    it 'corrects translated HTML' do
```

Comment:
This test does not what the description says. It does not verify that the result is the expected corrected translated HTML. Please add an expectation on `text` to verify that it holds the expected corrected translated HTML.